components/datatypes/consent-preferences.schema.json

```diff
@@ -0,0 +1,256 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/consents-and-preferences",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Privacy/Personalization/Marketing Preferences (Consent)",
+  "description": "This schema captures privacy, personalization and marketing preferences (consents).",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "definitions": {
+    "choice-value": {
+      "title": "Choice Value",
+      "description": "The value specified by the user as their preference or the basis of processing that precludes the need for user consent",
+      "type": "string",
+      "enum": ["y", "n", "p", "u", "LI", "CT", "CP", "VI", "PI"],
+      "meta:enum": {
+        "y": "Yes",
+        "n": "No",
+        "p": "Pending verification",
+        "u": "Unknown",
+        "LI": "Legitimate Interest",
+        "CT": "Contract",
+        "CP": "Compliance with a Legal Obligation",
+        "VI": "Vital Interest of the Individual",
+        "PI": "Public Interest"
+      }
+    },
+    "timestamp": {
+      "title": "Preference timestamp",
+      "description": "Time this specific preference was collected. This value should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\".",
```

Comment:
I know a bit obvious: "**Date and** time this specific preference was collected. This value should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."